modules/release-notes/con-relnotes-notable-features.adoc

```diff
@@ -6,8 +6,7 @@ This section highlights new features in {product} {product-version}.
 == {product} Operator is now generally available (GA)
 
 You can use the {product} Operator to install {product-short} on your {ocp-short} cluster.
-//For more information, see the link:{LinkAdminGuide}[Installing {product} using the Operator] section in the _Administration guide_.
-//replace with link to the installation guide.
+For more information, see the link:{installing-on-ocp-book-url}[Installing {product} using the Operator].
```

Comment:
This link should point to the 'Installing RHDH using the Operator' section, but it currently directs to the OCP guide. Could we please update the link to include the anchor for the Operator section?